src/app/containers/CmsHelpPage/style.scss

```diff
@@ -0,0 +1,70 @@
+@import 'variables.scss';
+
+:local {
+  .container {
+    min-width: $min-width-container;
+    width: 100%;
+    font-family: 'Noto Serif';
```

Comment:
Add the following code to `<head>` to make sure that the font is loaded (check `index.ejs`):

```
<link rel="preconnect" href="https://fonts.gstatic.com">
<link href="https://fonts.googleapis.com/css2?family=Noto+Serif:wght@400;700&display=swap" rel="stylesheet">
```

Add fallback option in case that the font is not loaded:

`font-family: 'Noto Serif', serif;`